### PR TITLE
DoubleDQN doesn't utilize DoubleQLearning

### PR DESCRIPTION
**Pull request by nishantkr18**
1) Changed config.DoubleQLearning() = true; for Double DQN 
2) Comment changes

### CODE REVIEW

**Comment by nishantkr18** [comment]
> Good catch.
> We had an issue where some of our automatic tests weren't running, but I believe it's now resolved. Can you [merge the latest master changes](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/merging-an-upstream-repository-into-your-fork) to try to get github to run those tests again, so we can check if this passes?

I have done as you said, but GitHub is not running the tests again. Maybe, I'll have to create a new PR? Is that correct, or am I missing something?

**Comment by kartikdutt18** [comment]
Hey @nishantkr18, You need to rebase your branch or merge master of mlpack/master. Once you do that the tests will run. Thanks a lot 👍.

**Comment by zoq** [comment]
Just manually started the pipeline: https://dev.azure.com/mlpack/mlpack/_build/results?buildId=1029&view=results

File: src/mlpack/tests/q_learning_test.cpp [hunk 1]
```diff
@@ -215,7 +215,7 @@ BOOST_AUTO_TEST_CASE(CartPoleWithDoubleDQN)
       averageReturn(episodeReturn);
 
       /**
-       * Reaching running average return 35 is enough to show it works.
+       * Reaching running average return 40 is enough to show it works.
```
Comment (zoq) [inline src/mlpack/tests/q_learning_test.cpp]: Oh, nice catch!

**Pull request merged**